test/integration_test.rb

```diff
@@ -229,6 +235,46 @@ def test_mixed_borders
     end
     @workbook.apply_styles
     assert_equal 8, @workbook.styled_cells.count
-    assert_equal 6, @workbook.style_index.count
+    assert_equal 6, @workbook.styles.style_index.count
+  end
+
+  def test_merge_styles_1
+    @filename = 'merge_styles_1'
+    @workbook.add_worksheet do |sheet|
+      sheet.add_row
+      sheet.add_row ['', '1', '2', '3']
+      sheet.add_row ['', '4', '5', '6']
+      sheet.add_row ['', '7', '8', '9']
+      sheet.add_style 'B2:D4', b: true
+      sheet.add_style 'B2:D4', border: { style: :thin, color: '000000' }
+    end
+    @workbook.apply_styles
```

Comment:
Could you replace this line with `serialize(filename)` and replace `@filename` on line 242 with `filename`?

I forgot to remove `@filename` in all places when I was removing tests "teardown" hook that was using the variable. The reason I'm asking to add `serialize` is for visual inspection of test results in the `tmp` folder.